include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -1009,7 +1009,7 @@ struct OrtApi {
    * This function only works for numeric tensors.
    * This is a no-copy method whose pointer is only valid until the backing OrtValue is free'd.
    */
-  ORT_API2_STATUS(TensorAt, _Inout_ OrtValue* value, size_t* location_values, size_t location_values_count, _Outptr_ void** out);
+  ORT_API2_STATUS(TensorAt, _Inout_ OrtValue* value, const int64_t* location_values, size_t location_values_count, _Outptr_ void** out);
```

Comment:
This is an ABI change.

Reply:
I believe this was not released yet.

---
In reply to: [488958877](https://github.com/microsoft/onnxruntime/pull/5180#discussion_r488958877) [](ancestors = 488958877)